src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PeopleLink } from '../PeopleLink';
+import { ParentsLink } from '../ParentsLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => (
```

Comment:
If you don't pass the entire `person` object anywhere, it can be destructuring.

Reply:
![image](https://github.com/mate-academy/react_people-table-basics/assets/43435306/86123fff-4490-47bf-a283-38d65dd28ab2)

Reply:
@Wesses yes you are right 😄 
But then you can do descrutization not as an argument, but already in the body of the function